libraries/standard/mqtt/include/iot_mqtt.h

```diff
@@ -807,8 +807,7 @@ const char * IotMqtt_OperationType( IotMqttOperationType_t operation );
  * @param[in] pTopicFilter The topic filter to check.
  * @param[in] topicFilterLength Length of `pTopicFilter`.
  * @param[out] pCurrentSubscription If a subscription is found, its details are
- * copied here. This output parameter is only valid if this function returns `true`.
- * Pass `NULL` to ignore.
+ * copied here. This output parameter is only valid if this function returns `true` (NULL to disable)
```

Comment:
Nit: period at end of comment, use code syntax for NULL: (`NULL` to disable).

Reply:
Added period here and changed to `NULL` throughout.